### PR TITLE
2022-10-17 Release Notes

### PR DESCRIPTION
**Pull request by CocoWang-wql**
2022-10-17 Release Notes

### CODE REVIEW

**Review by kaarthis — CHANGES_REQUESTED** [review]
If there are links for the behavior changes it would be very handy for the customers.

**Comment by CocoWang-wql** [comment]
> If there are links for the behavior changes it would be very handy for the customers.

I added the rename link. For removing `cloud-node-manager` CPU limit, as confirmed with Pengfei, there is no link to demonstrate.

**Review by miwithro — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+* Some AKS labels are being deprecated with the Kubernetes 1.26 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation. `beta.kubernetes.io/arch=` and `beta.kubernetes.io/os=`  are still applied by kubelet in kubernetes code.
+* Docker is no longer supported as a container runtime on Windows. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+### Release notes
+
+* Features
+  * [OIDC Issuer](https://learn.microsoft.com/en-us/azure/aks/cluster-configuration#oidc-issuer) is now Generally Available.
+* Behavior Changes
+  * The CPU limits for `cloud-node-manager` has been removed.
```
Comment (dennis-menge) [inline CHANGELOG.md:20]: `has` -> `have`

**Review by kaarthis — APPROVED** [review]

**Comment by kaarthis** [comment]
Approved @CocoWang-wql

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+  * OMSAgent resources will be renamed to [`ama-logs`](https://techcommunity.microsoft.com/t5/azure-monitor-status-archive/name-update-for-agent-and-associated-resources-in-azure-monitor/ba-p/3576810).
+  * `ip-masq-agent-v2` is updated to [v0.1.5](https://github.com/Azure/ip-masq-agent-v2/releases/tag/v0.1.5), which includes the usage of a distroless-iptables image and a reduction in image size from 75.4MB to 34.2MB.
```
Comment (palma21) [inline CHANGELOG.md:22]: These are 2 things, one behavior change and 1 component update

File: CHANGELOG.md (hunk 1 above)
Comment (palma21) [inline CHANGELOG.md:20]:
```suggestion
  * The CPU limits for `cloud-node-manager` have been removed.
```

**Pull request merged**